### PR TITLE
Add section to GitOps page explaining web UI restrictions

### PR DESCRIPTION
**Pull request by marians**
### What does this PR do?

A section is added to the page https://docs.giantswarm.io/advanced/gitops/ which explains how resources that are managed through GitOps/FluxCD are restricted from editing/deletion via the web UI.

### What is the effect of this change to users?

Users following a "Learn more" link from the web UI will find information explaining what the indicator they see means.

Users reading the page from top to bottom will learn that GitOps managed resources are restricted from edits/deletion in the web UI.

### How does it look like?

(Please add screenshots or screen recordings demonstrating the change.)

### Any background context you can provide?

- https://github.com/giantswarm/roadmap/issues/624

### CODE REVIEW

**Comment by marians** [comment]
This is now ready to be merged, as the happa change will happen soon IMO.

File: src/content/advanced/gitops/index.md [hunk 1]
```diff
@@ -72,6 +72,26 @@ In order to follow the [Watching for new commits](#watching-for-new-commits) sec
...
+
+- **Organizations** managed by FluxCD cannot be deleted via the web UI.
+
+- **Clusters** managed by FluxCD cannot be edited, upgraded, or deleted via the web UI.
+
+- **Node pools** managed by FluxCD cannot be edited or deleted via the web UI.
+
+- **Apps** managed by FluxCD cannot be edited or uninstalled via the web UI.
+
+As a signal that a resource is managed through FluxCD, the web UI relies on the resource label TODO.
```
Comment (kuosandys) [inline src/content/advanced/gitops/index.md:93]: I think this TODO was leftover from earlier - could we add the label here?

Comment (marians) [reply src/content/advanced/gitops/index.md:93]: Ooops, good catch. I'll add the info on the two labels.

Comment (marians) [reply src/content/advanced/gitops/index.md]: Done

**Review by kuosandys — APPROVED** [review]

**Pull request merged**